cfme/tests/cloud/test_provisioning.py

```diff
@@ -125,35 +125,47 @@ def vm_name(request):
     return random_vm_name('prov')
 
 
+@pytest.fixture(scope="function")
+def provision_check(request, provider):
```

Comment:
I don't see where ``request`` is being used.

Reply:
corrected